ch_backup/logic/lock_manager.py

```diff
@@ -44,20 +47,24 @@ def __init__(self, lock_conf: dict, zk_ctl: Optional[ZookeeperCTL]) -> None:
         self._disabled = False
         self._lock_timeout = lock_conf.get("lock_timeout")
 
-    def __call__(self, distributed: bool = True, disabled: bool = False):  # type: ignore
+    def __call__(self, distributed: bool = True, disabled: bool = False, operation: str = "UNKNOWN"):  # type: ignore
         """
         Call-method for context manager
         """
         self._distributed = distributed
         self._disabled = disabled
+        self._lock_id = f"{operation}/{socket.getfqdn()}"
+        self._logger = ChBackupLogger("lock-man", lock_id=self._lock_id)
```

Comment:
In my opinion it is over engeneering i would propose to just something like this. This will significantly reduce the PR.
Initially in my comment I confused `opt()` and `patch()`. Both methods return a custom logger.

```
def log_message(record):
    record['message'] = f"Changed message: {record['message']}"

self._logger = logger.patch(log_message)
self._logger.info('Hello')  # will output `Changed message: Hello``
```